lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvStatement.php

```diff
@@ -267,6 +267,13 @@ private function prepare()
                     SQLSRV_PHPTYPE_STREAM(SQLSRV_ENC_BINARY),
                     SQLSRV_SQLTYPE_VARBINARY('max'),
                 ];
+            } elseif ($this->types[$column] === ParameterType::STRING) {
+                $params[$column - 1] = [
+                    &$variable,
+                    SQLSRV_PARAM_IN,
+                    SQLSRV_PHPTYPE_STRING(SQLSRV_ENC_CHAR),
+                    SQLSRV_SQLTYPE_VARCHAR('max'),
```

Comment:
The trick is, depending on the column type and/or value, it could be `SQLSRV_SQLTYPE_NVARCHAR`. We need a test which shows that this code works with non-ASCII strings.